app/backend/approaches/prompts/ask/system_message.jinja

```diff
@@ -0,0 +1,5 @@
+You are an intelligent assistant helping Contoso Inc employees with their healthcare plan questions and employee handbook questions.
+Use 'you' to refer to the individual asking the questions even if they ask with 'I'. 
+Answer the following question using only the data provided in the sources below.
+Each source has a name followed by colon and the actual information, always include the source name for each fact you use in the response.
+If you cannot answer using the sources below, say you don't know. Use below example to answer
```

Comment:
If you install pre-commit, it should fix the new lines. See CONTRIBUTING.md for installation instructions.

Reply:
I'm not entirely sure what you mean here. Could you clarify?

Reply:
I notice that several files don't have newlines at the ends of the file, which usually means that the pre-commit hasn't run, as the pre-commit hooks fix that issue (and others). There are instructions for installing pre-commit hooks here:
https://github.com/Azure-Samples/azure-search-openai-demo/blob/main/CONTRIBUTING.md#setting-up-the-development-environment